utils/logger.js

```diff
@@ -0,0 +1,107 @@
+const LogLevel = {
+  LOG: {
+    value: 0,
+    method: console.log,
+  },
+  INFO: {
+    value: 1,
+    method: console.info,
+  },
+  DEBUG: {
+    value: 2,
+    method: console.debug,
+  },
+  WARN: {
+    value: 3,
+    method: console.warn,
+  },
+  ERROR: {
+    value: 4,
+    method: console.error,
+  },
+};
+
+class Logger {
+  constructor(scope, level) {
+    this.level = +level || LogLevel.ERROR.value;
```

Comment:
Why are we setting default as error level?

Reply:
To avoid noise in the browser console.